client/router.js

```diff
@@ -376,15 +374,11 @@ class Router {
       subscriptions = _.values(globalRoute.getAllSubscriptions());
       subscriptions = subscriptions.concat(_.values(currentRoute.getAllSubscriptions()));
     } else {
-      subscriptions = _.map(args, (subName) => {
-        return globalRoute.getSubscription(subName) || currentRoute.getSubscription(subName);
-      });
+      subscriptions = _.map(args, (subName) => globalRoute.getSubscription(subName) || currentRoute.getSubscription(subName));
```

Comment:
not really fan of one-liners, came for line :381 , is there any real benefits behind it?

Reply:
see above. will undo.